resources/lang/en/manager/application_post.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+return [
+    /*
+    * --------------------------------------------------------------------------
+    * Manager Application Post View Language Lines
+    * --------------------------------------------------------------------------
+    * controller: Controllers/ApplicationController.php
+    * view: views/manager/application_post.html.twig
+    * route: /manager/applications/#
+    *
+    *
+    */
+
+    'title' => 'Applicant Profile',
+    'header' => [
+        'twitter_title' => 'Visit :name\'s Twitter profile.',
+        'linkedin_title' => 'Visit :name\'s LinkedIn profile.'
+    ],
+    'content' => [
+        'job_poster' => 'Job Poster',
+        'review' => [
+            'name' => 'Name',
+            'info' => 'Info',
+            'reviewed' => 'Reviewed?tasitc',
```

Comment:
Looks like a typo here and on line 28.

Reply:
Yeah, that wasn't suppose to make it into the commit.